examples/I/hashmap.c

```diff
@@ -0,0 +1,132 @@
+#include <string.h>
+
+#include "hashmap.h"
+
+struct hashmap_t
+{
+    int size;
+    struct alloc_t alloc;
+    struct hashmap_keyval_t** buckets;
+};
+
+struct hashmap_keyval_t
+{
+    char* key;
+    value_t val;
+    struct hashmap_keyval_t* next;
+};
+
+
+static int hash_func(char* key)
+{
+	int len = strlen(key);
```

Comment:
Coding style - additional whitespace

Reply:
Tab/spaces mismatch in fact, not an unnecessary whitespace
```suggestion
    int len = strlen(key);
```

Reply:
done